app/helpers/react_on_rails_helper.rb

```diff
@@ -76,24 +76,28 @@ def render_js(js_expression)
 
   private
 
-  def debug_js(react_component, data_variable, dom_id, trace)
+  def debug_js(react_component_name, data_variable, dom_id, trace)
     if trace
       <<-JS.strip_heredoc
-        console.log("CLIENT SIDE RENDERED #{react_component} with dataVariable #{data_variable} to dom node with id: #{dom_id}");
+        console.log("CLIENT SIDE RENDERED #{react_component_name} with data_variable #{data_variable} to dom node with id: #{dom_id}");
       JS
     else
       ""
     end
   end
 
-  def define_render_if_dom_node_present(react_component, data_variable, dom_id, trace)
+  def render_js_react_element(react_component_name, props_name)
```

Comment:
Let's change `props_name` to `props_string` b/c it's either a name or the full json version of a hash.